### PR TITLE
feat(PanoViewer): add useGyro options

### PR DESCRIPTION
**Pull request by happyhj**
## Issue
#105

## Detail 
* add useGyro options to PanoViewer
* apply useGyro options to Manual test

### CODE REVIEW

File: src/PanoViewer/PanoViewer.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ export default class PanoViewer extends Component {
 	 * @param {Boolean} [config.showPolePoint=false] If false, the pole is not displayed inside the viewport <ko>false 인 경우, 극점은 뷰포트 내부에 표시되지 않습니다</ko>
 	 * @param {Boolean} [config.useZoom=true] When true, enables zoom with the wheel and Pinch gesture <ko>true 일 때 휠 및 집기 제스춰로 확대 / 축소 할 수 있습니다.</ko>
 	 * @param {Boolean} [config.useKeyboard=true] When true, enables the keyboard move key control: awsd, arrow keys <ko>true 이면 키보드 이동 키 컨트롤을 활성화합니다: awsd, 화살표 키</ko>
+	 * @param {String} [config.useGyro=yawPitch] Enables control through device motion. ("none", "yawPitch") <ko>디바이스 움직임을 통한 컨트롤을 활성화 합니다. ("none", "yawPitch") </ko>
```
Comment (jongmoon) [inline src/PanoViewer/PanoViewer.js]: @happyhj Suggest
`
[config.useGyro="yawPitch"] or [config.useGyro=PanoViewer.GYRO_MODE.YAW_PITCH]
`

File: src/YawPitchControl/YawPitchControl.js [hunk 2]
```diff
@@ -228,6 +231,16 @@ const YawPitchControl = class YawPitchControl extends Component {
 			}
 		}
 
+		if (keys.some(key => key === "useGyro") && this.axesTiltMotionInput) {
+			const useGyro = this.options.useGyro;
+
+			if (useGyro === GYRO_MODE.YAWPITCH) {
+				this.axes.connect(["yaw", "pitch"], this.axesTiltMotionInput);
+			} if (useGyro === GYRO_MODE.NONE) {
```
Comment (jongmoon) [inline src/YawPitchControl/YawPitchControl.js]: @happyhj Use `else if` instead.

File: src/YawPitchControl/YawPitchControl.js [hunk 3]
```diff
@@ -228,6 +231,16 @@ const YawPitchControl = class YawPitchControl extends Component {
 			}
 		}
 
+		if (keys.some(key => key === "useGyro") && this.axesTiltMotionInput) {
+			const useGyro = this.options.useGyro;
+
+			if (useGyro === GYRO_MODE.YAWPITCH) {
+				this.axes.connect(["yaw", "pitch"], this.axesTiltMotionInput);
```
Comment (jongmoon) [inline src/YawPitchControl/YawPitchControl.js]: @happyhj Is this not problem to connect again if already connected?

Comment (happyhj) [reply src/YawPitchControl/YawPitchControl.js]: If the useGyro value changes, the row will be executed, so there is no chance of duplicate execution.

File: src/PanoViewer/PanoViewer.js (hunk 1 above)
Comment (happyhj) [reply src/PanoViewer/PanoViewer.js]: we don't need double quote to default value of param 

[ref](http://usejsdoc.org/tags-param.html)
```
/**
 * @param {string} [somebody=John Doe] - Somebody's name.
 */
function sayHello(somebody) {
    if (!somebody) {
        somebody = 'John Doe';
    }
    alert('Hello ' + somebody);
}
```

**Pull request merged**